manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: eFMer.BoincTasks
+PackageVersion: 1.7.8

```

```suggestion
PackageVersion: 1.78
```
This needs to match ARP